vue/components/ui/generic-button-platforme/generic-button-platforme.vue

```diff
@@ -0,0 +1,109 @@
+<template>
+    <button
+        class="button button-content"
+        v-bind:class="{
+            arrow: arrowAnimation,
+            scale: scaleAnimation
+        }"
+        v-bind:style="{
+            'background-color': backgroundColor,
+            'color': textColor,
+            'border-color': borderColor
+        }"
+    >
+        {{ text }}
+    </button>
+</template>
+
+<style scoped>

```

Comment:
Organize the selectors from the least specific (button) to the most specific (.button-content.arrow:hover:not(:disabled)::after)